test/e2e/vsphere/multi-nic.go

```diff
@@ -130,7 +130,7 @@ func failIfMachinesDoNotHaveAllPortgroups(platformSpec configv1.PlatformSpec, ma
 
 func failIfMachineDoesNotHaveAllPortgroups(machine machinev1beta1.Machine, failureDomain configv1.VSpherePlatformFailureDomainSpec) {
 
-	By("checking to see if machine has all portgroups")
+	By(fmt.Sprintf("checking to see if machine %s has all portgroups", machine.Name))
```

Comment:
Nit: What I learned recently is steps (`By`), in general, are statically and variants would be used printing methods (`framework.Logf`, `fmt.Print*`, etc)